internetarchive/openlibrary
# Update to webpack 5

No issue :)


fix
### Technical


### Testing

1) build assets (dev and prod) Note: no default set! Set explicitly for certainty when build
2) check whether they are acually compiled

Build log for local machine:
https://gist.github.com/jimman2003/04a105c78b50e7440c6e7656dcbd00cc
### Screenshot


### Stakeholders

@cdrini ,@jdlrobson
